02-project-intro.Rmd

```diff
@@ -163,6 +163,13 @@ Now we have a good directory structure we will now place/save the data file in t
 >
 > We will load and inspect these data later.
 
+> ## Challenge 2 {.challenge}
+> It is usefull to get some general idea about the dataset, directly from the
```

Comment:
Still need to fix the spelling of 'usefull'. I agree with @tomwright01's general sense that more flexibility is better with regard to tool choice, but I also recognize that other parts of this very same lesson discuss creating a `.gitignore` file, for example, which would only happen after a decent amount of time working with shell and git. 

It seems to me that a general principle should be making the lessons as modular as possible but not excessively so.